test-suite/js/src/test/scala/org/scalajs/testsuite/compiler/InteroperabilityTest.scala

```diff
@@ -182,6 +182,59 @@ class InteroperabilityTest {
     assertEquals(10, obj6.x)
   }
 
+  @Test def nestedNativeJSClasses(): Unit = {
+    js.eval("""
+      var InteroperabilityTestContainerClass = (function(a) {
```

Comment:
Hum, could you elaborate? I don't understand what you mean by "we don't copy it".

Reply:
I meant to make sure we don't copy the outer class instead of having a reference to it. But on second thought that's probably overkill. Seems more of a C++ like problem to have :)